smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -48,6 +48,7 @@ be used.
                             Sockets, otherwise N/A
                             switch form uses local IP address as hostname
   --username                anonymous, or Key Vault if --registry is specified
+  --no-deployment           Start IoT Edge modules deployment (EdgeHub and custom modules)
```

Comment:
The first word ("start") should **not** be capitalized, to be consistent with the rest.

Reply:
...and `EdgeHub` should have a space, i.e., `Edge Hub`.

Reply:
I'd vote for text like this: `Deploy Edge Hub and temperature sensor modules`